docs/core/preview3/tools/csproj.md

```diff
@@ -136,34 +197,91 @@ The following example specifies the fallbacks only for the `netcoreapp1.0` targe
 ```
 
 ## NuGet metadata properties
-With the move to MSbuild, we have moved the input metadata that is used when packing a NuGet package from project.json to csproj files. The inputs are MSBuild properties. The following is the list of properties that are used as inputs to the packing process when using the `dotnet pack` command or the `Pack` MSBuild target that is part of the SDK. 
-
-* IsPackable
-* PackageVersion
-* PackageId
-* Title
-* Authors
-* Description
-* Copyright
-* PackageRequireLicenseAcceptance
-* PackageLicenseUrl
-* PackageProjectUrl
-* PackageIconUrl
-* PackageReleaseNotes
-* PackageTags
-* PackageOutputPath
-* IncludeSymbols
-* IncludeSource
-* PackageTypes
-* IsTool
-* RepositoryUrl
-* RepositoryType
-* NoPackageAnalysis
-* MinClientVersion
-* IncludeBuildOutput
-* IncludeContentInPack
-* BuildOutputTargetFolder
-* ContentTargetFolders
-* NuspecFile
-* NuspecBasePath
-* NuspecProperties
\ No newline at end of file
+With the move to MSbuild, we have moved the input metadata that is used when packing a NuGet package from project.json to csproj files. The inputs are MSBuild properties so they have to go within a `<PropertyGroup>` group. The following is the list of properties that are used as inputs to the packing process when using the `dotnet pack` command or the `Pack` MSBuild target that is part of the SDK. 
+
+### IsPackable
+TBD
+
+### PackageVersion
+Specified a version that the resulting package will have. Accepts all forms of NuGet version string. 
+
+### PackageId
+Specify a name for the resulting package. If not specified, the `pack` operation will default to using the AssemblyName or directory name as the name of the package. 
+
+### Title
+A human-friendly title of the package, typically used in UI displays as on nuget.org and the Package Manager in Visual Studio. If not specified, the package ID is used instead.
+
+### Authors
+A comma-separated list of packages authors, matching the profile names on nuget.org. These are displayed in the NuGet Gallery on nuget.org and are used to cross-reference packages by the same authors.
+
+### Description
+A long description of the package for UI display.
+
+### Copyright
+Copyright details for the package.
+
+### PackageRequireLicenseAcceptance
+A Boolean value specifying whether the client must prompt the consumer to accept the package license before installing the package.
+
+### PackageLicenseUrl
+An URL to the license that is applicable to the package.
+
+### PackageProjectUrl
+A URL for the package's home page, often shown in UI displays as well as nuget.org.
+
+### PackageIconUrl
+A URL for a 64x64 image with transparenty background to use as the icon for the package in UI display.
+
+### PackageReleaseNotes
+Release notes for the package.
+
+### PackageTags
+A list of tags that designates the package. The list is represented as a list of tags that are comma-delimited. 
+
+### PackageOutputPath
+TBD
+
+### IncludeSymbols
```

Comment:
ends up creating a separate nuget package with a .symbols.nupkg extension and puts the PDB next to the DLL